app/services/appropriate_bodies/claim_an_ect/revert_claim.rb

```diff
@@ -0,0 +1,26 @@
+module AppropriateBodies
+  module ClaimAnECT
+    class RevertClaim
+      attr_reader :appropriate_body, :teacher, :author, :induction_period
+
+      def initialize(appropriate_body:, teacher:, author:, induction_period:)
+        @appropriate_body = appropriate_body
+        @teacher = teacher
+        @author = author
+        @induction_period = induction_period
+      end
+
+      def revert_claim
+        ActiveRecord::Base.transaction do
+          induction_period.destroy!
+          ResetInductionJob.perform_later(trn: teacher.trn)
```

Comment:
If the ECT has any induction periods at all they're `InProgress` from TRS's point of view, so we should only reset it if we're deleting their only `InductionPeriod`.